### PR TITLE
feature: update readme

### LINKED ISSUE #134: docs: update README.md and README.nob.md to reflect current state of affairs
The README hasn't been kept updated -- we now load in significantly more data to the client, for one.
- [ ] Update README.md
- [ ] Update README.nob.md

### PR DESCRIPTION
**Pull request by Log234**
# Description
Updates the readme files with the changes since they were last updated.

Closes #134

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Lavinia-client
 
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/64b563a2c5e349e2a13fcfe3c1bc1008)](https://www.codacy.com/gh/Project-Lavinia/Lavinia-client?utm_source=github.com&utm_medium=referral&utm_content=Project-Lavinia/Lavinia-client&utm_campaign=Badge_Grade) [![Total alerts](https://img.shields.io/lgtm/alerts/g/Project-Lavinia/Lavinia-client.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/Project-Lavinia/Lavinia-client/alerts/)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/64b563a2c5e349e2a13fcfe3c1bc1008)](https://www.codacy.com/gh/Project-Lavinia/Lavinia-client?utm_source=github.com&utm_medium=referral&utm_content=Project-Lavinia/Lavinia-client&utm_campaign=Badge_Grade) [![Language grade: JavaScript](https://img.shields.io/lgtm/grade/javascript/g/Project-Lavinia/Lavinia-client.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/Project-Lavinia/Lavinia-client/context:javascript)
```
Comment (simendag) [inline README.md:3]: Shouldn't there be three badges? Codacy rating,, LGTM alerts, and LGTM rating?

File: README.nob.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Lavinia-client
 
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/3ab572caf1584cd6b6f9c98140404fd8)](https://www.codacy.com/manual/Project-Lavinia/Lavinia-client?utm_source=github.com&utm_medium=referral&utm_content=Project-Lavinia/Lavinia-client&utm_campaign=Badge_Grade) [![Total alerts](https://img.shields.io/lgtm/alerts/g/Project-Lavinia/Lavinia-client.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/Project-Lavinia/Lavinia-client/alerts/)
+[![Codacy Badge](https://app.codacy.com/project/badge/Grade/64b563a2c5e349e2a13fcfe3c1bc1008)](https://www.codacy.com/gh/Project-Lavinia/Lavinia-client?utm_source=github.com&utm_medium=referral&utm_content=Project-Lavinia/Lavinia-client&utm_campaign=Badge_Grade) [![Language grade: JavaScript](https://img.shields.io/lgtm/grade/javascript/g/Project-Lavinia/Lavinia-client.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/Project-Lavinia/Lavinia-client/context:javascript)
```
Comment (simendag) [inline README.nob.md:3]: See comment on English README.md

File: README.nob.md [hunk 3]
```diff
@@ -10,9 +10,15 @@ Denne guiden på andre språk:
 
 En applikasjon som simulerer mandatdistribusjon i valg. Nåværende funksjonalitet inkluderer:
 
--   Stortingsvalgene i 2017, 2013 og 2009
--   Fordelingsmetodene Sainte-Lagüe (modifisert) og D'Hondt
+-   Alle stortingsvalg siden 1945
+-   Fordelingsmetodene Sainte-Lagüe (modifisert), d'Hondt, største brøks metode (Hare, Droop & Hagenbach-Bischoff)
 -   Detaljerte tabeller som beskriver simulasjonsresultatene
+-   Parametere for å justere hvordan mandatene distribueres
```
Comment (simendag) [inline README.nob.md:16]:
```suggestion
-   Parametere for å justere hvordan mandatene fordeles
```
might be better

File: README.nob.md [hunk 4]
```diff
@@ -10,9 +10,15 @@ Denne guiden på andre språk:
...
+-   Sammenslåing av fylker for å simulere effekten av de nye fylkene i 2020
```
Comment (simendag) [inline README.nob.md:17]:
```suggestion
-   Sammenslåing av fylker for å simulere effekten av de nye regionene i 2020
```
might be an improvement

File: README.md (hunk 1 above)
Comment (Log234) [reply README.md:3]: Sure, we can keep the number of alerts.

File: README.nob.md (hunk 4 above)
Comment (Log234) [reply README.nob.md:17]: They are counties, rather than regions. Isn't it more correct to use the proper word for it?

Comment (simendag) [reply README.nob.md:17]: It is more clear it has to do with the region reform this way (and it is unknown whether the elections will work this way in the future).

**Comment by Log234** [comment]
Does anyone have time to review this PR?

**Pull request merged**